### ISSUE TITLE
Allow different patterns for the `t.try()` callback argument

### ISSUE THREAD

**Issue by novemberborn** [avajs/eslint-plugin-ava#273]
https://github.com/avajs/ava/pull/1947 landed a new `t.try()` assertion. It's experimental, one of the reasons for which is that it advocates some new usages of AVA that this plugin rejects.

`t.try()` takes a callback, which receives another execution object:

```js
const attempt = await t.try(t => {
  t.pass()
})

attempt.commit()
```

Not all linting setups allow you to shadow variables like this. I'm proposing we also allow `tt`, and `t1` through `t9`. Thoughts?

**Comment by sindresorhus** [comment]
I would also allow `t_`. I usually postfix with underscore for shadowed variables with the same name.

**Comment by novemberborn** [comment]
Yes. Looks like you're agreed with the other patterns so then we need to add support for all of these.
